norns/maiden.md

```diff
@@ -70,38 +70,50 @@ Each entry has an **INSTALL** action, which can be used to install the selected
 You will have to restart norns if a freshly-installed project contains an engine
 {: .label}
 
-### contribute
+### contribute & gather
 
-To add to the [community project repo](https://github.com/monome/norns-community/blob/master/community.json), please submit a pull request with the following information:
+The norns ecosystem was created with community as a focus. The exchange of ideas leads to new ideas.
+
+As such, you are encourage to share your creations with the community.
+
+The first step is to host your script on a public git repository (e.g. on GitHub, GitLab ...).
```

Comment:
suggest not being semi-prescriptive here and also mentioning the zip option if that is desirable. historically git was seen as a barrier for entry but maybe that view is no longer held?